lib/common.bash

```diff
@@ -234,6 +234,7 @@ clean_env_ctr()
 	for i in "${containers[@]}"; do
 		task="$(sudo ctr task ls | grep -E "\<${i}\>" | cut -f1 -d" " || true)"
 		[ -n "${task}" ] && sudo ctr task kill "${task}"
+		sudo ctr tasks kill -s SIGKILL "${task}"
```

Comment:
Should add a pre-check `[ -n ${taks} ] `like the line above?

Reply:
@liubin changes applied, thanks